srcpkgs/asterisk/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'asterisk'
+pkgname=asterisk
+version=19.5.0
+revision=1
+build_style=gnu-configure
+conf_files="/etc/asterisk/*.conf"
+make_dirs="/var/log/asterisk 0755 _asterisk _asterisk
+ /var/cache/asterisk 0755 _asterisk _asterisk
+ /var/lib/asterisk 0755 _asterisk _asterisk
+ /var/spool/asterisk 0755 _asterisk _asterisk"
+hostmakedepends="curl tar pkg-config"
+makedepends="openssl-devel libedit-devel libuuid-devel jansson-devel
+ libxml2-devel sqlite-devel speex-devel libvorbis-devel libgsm-devel
+ alsa-lib-devel libbluetooth-devel libsrtp-devel"
+short_desc="Complete PBX solution"
+maintainer="teldra <teldra@rotce.de>"
+license="custom:Proprietary GPL-2.0-or-later"
```

Comment:
```suggestion
license="custom:Proprietary, GPL-2.0-or-later"
```

Reply:
> Asterisk is distributed under the GNU General Public License version 2
> and is also available under alternative licenses negotiated directly
> with Digium, Inc. If you obtained Asterisk under the GPL, then the GPL
> applies to all loadable Asterisk modules used on your system as well,
> except as defined below. The GPL (version 2) is included in this
> source tree in the file COPYING.
> 
> This package also includes various components that are not part of
> Asterisk itself; these components are in the 'contrib' directory
> and its subdirectories. These components are also distributed under the
> GPL version 2 as well.

What part of it is proprietary? (also, if there are proprietary bits, it should probably also have `repo=nonfree`)

Reply:
Based on your quote, only `GPL-2.0-or-later` left in template.

Reply:
alpine lists it as `GPL-2.0-only WITH OpenSSL-Exception`, which i think matches what's in the LICENSE file

Reply:
weird, seems that exception doesn't have an spdx name

Reply:
Should we ignore this for now, because this seems to be an ongoing issue: https://github.com/spdx/license-list-XML/issues/1356 ?

Reply:
WITH is parsed as a comma in xlint, so we could do custom:OpenSSL-Exception I guess

Reply:
Yes. Thank you!